CMakeLists.txt

```diff
@@ -63,15 +69,17 @@ add_library(usgscsm SHARED
             src/UsgsAstroLsSensorModel.cpp
             src/UsgsAstroSarSensorModel.cpp
             src/Distortion.cpp
-            src/Utilities.cpp)
+            src/Utilities.cpp
+            src/EigenUtilities.cpp)
 
 set_target_properties(usgscsm PROPERTIES
     VERSION ${PROJECT_VERSION}
     SOVERSION 1
 )
 
 set(USGSCSM_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/include/usgscsm"
-                         "${CMAKE_CURRENT_SOURCE_DIR}/include/")
+                         "${CMAKE_CURRENT_SOURCE_DIR}/include"
+                         "${EIGEN3_INCLUDE_DIR}")
```

Comment:
Link Eigen as a target instead of setting variables like this:

https://eigen.tuxfamily.org/dox/TopicCMakeGuide.html

Reply:
Eigen also needs to be added to the environment file

Reply:
Added.

Reply:
Linking Eigen is not the issue. It is looking up the header which is a problem. I tried yesterday along the lines of what you suggest and it was not picking up the include directory. The way I put it above works both with internal and external dependencies.